src/pcb/pcb_manual_edit_conflict_warning.ts

```diff
@@ -0,0 +1,44 @@
+import { z } from "zod"
+import { getZodPrefixedIdWithDefault } from "../common"
+import { expectTypesMatch } from "../utils/expect-types-match"
+
+export const pcb_manual_edit_conflict_warning = z
+  .object({
+    type: z.literal("pcb_manual_edit_conflict_warning"),
+    pcb_warning_id: getZodPrefixedIdWithDefault(
```

Comment:
The id must always match the type (can you also fix this for the schematic warning if it's incorrect?) 

```suggestion
    pcb_manual_edit_conflict_warning_id: getZodPrefixedIdWithDefault(
```